packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.spec.ts

```diff
@@ -61,8 +60,8 @@ before(() => {
         bind(QuickCommandService).toConstantValue({} as any);
         // tslint:disable-next-line:no-any mock TabBarToolbarRegistry
         bind(TabBarToolbarRegistry).toConstantValue({} as any);
-        // tslint:disable-next-line:no-any mock PluginSharedStyle
-        bind(PluginSharedStyle).toConstantValue({} as any);
+        // tslint:disable-next-line:no-any mock SharedStyle
+        bind(SharedStyle).toConstantValue({} as any);
```

Comment:
this line looks strange no ?

Reply:
Sorry @benoitf why do you find it strange?
Do you feel like `SharedStyle` previously `PluginSharedStyle` can be removed as a mock from the tests?

Reply:
to be more concrete, we shouldn't bind the core SharedStyle in plugin-ext

Reply:
no I read it too fast, it was in a test, I thought it was in plugin's core.